### ISSUE TITLE
A serious problem with Pandoc's lua script stopped the document from being compiled into DOCX

### ISSUE THREAD

**Issue by williamlorder** [quarto-dev/quarto-cli#1206]
### Bug description

In the latest version of quarto,when I compile the documentation using Lua scripts as a filter, I get the following error.This is a very important filter in scientific writing (Better Bibtex) that helps us to change plain text literature into literature in Word. You can view this filter below(https://retorque.re/zotero-better-bibtex/exporting/zotero.lua). I don't know if it was a problem with the filter or with quarto, but the problem was solved when I went back to version 0.9.551.

```lua
zotero-live-citations f34bfc770
Error running filter /Users/XXX/zotero.lua:
/Users/XXX/zotero.lua:1891: attempt to call a nil value (field 'next_id')
stack traceback:
```

### Checklist

- [X] [formatted your issue](https://yihui.org/issue/#please-format-your-issue-correctly) so it is easier for us to read?
- [X] included a minimal, self-contained, and reproducible example?
- [X] documented the quarto version you're running, by providing the output produced by `quarto check` in a terminal in your issue?
- [X] documented the RStudio IDE version you're running (if applicable), by providing the value displayed in the "About RStudio" main menu dialog?
- [X] documented which operating system you're running? If on Linux, please provide the specific distribution as well.
- [X] upgraded to the latest version, including your versions of R, the RStudio IDE, and relevant R packages?

**Comment by dragonstyle** [comment]
Thanks for reporting this. Can you share a sample `qmd` file that reproduces the issue?

**Comment by dragonstyle** [comment]
Thanks for letting us know about this - we changed the structure of our LUA filters inadvertently introduced a conflicting json implementation into the global namespace. I've fixed that here: f8fb41dac5aa9299873edd8f2a2cfb7cb579f5be

A fresh release should be available shortly.
